-- Avoid a GN "Assignment had no effect" error for the `inputs` variable
-- that is set but not consumed by this FreeBSD build.
--- build/toolchain/gcc_toolchain.gni.orig	2026-09-24 10:43:42 UTC
+++ build/toolchain/gcc_toolchain.gni
@@ -412,13 +412,6 @@ template("single_gcc_toolchain") {
         command = "cmd /s /c \"\"$python_path\" $tool_wrapper_path delete-file {{output}} && $command\""
       } else {
         command = "rm -f {{output}} && $command"
-
-        # Add ar to inputs if it's not from $PATH. Some toolchains leave
-        # |ar| empty during toolchain setup, so guard get_path_info() here.
-        if (ar != "" && get_path_info(ar, "file") != ar) {
-          inputs =
-              [ get_path_info(rebase_path(ar, ".", root_out_dir), "abspath") ]
-        }
       }
 
       # Almost all targets build with //build/config/compiler:thin_archive which
